react/week2/exersice-react/src/components-todo/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import TodoItems from './TodoItems'
+import TodoListItem from './TodoListItem'
+
+function TodoList() {
+    const originalTodos = TodoItems();
+    let [updatedTodos, setTodos] = useState(originalTodos);
+
+    let newTodoDescription = '';
+
+    function addNewTodo() {
+        const id = updatedTodos.length + 1;
+        const date = new Date().toLocaleDateString("en-US", { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' });
+        const newTodo = { id: id, description: newTodoDescription, date: date }
+        updatedTodos = updatedTodos.concat(newTodo);
+        setTodos(updatedTodos)
+    }
+
+    return (<div>
+        <input type='text' onChange={e => newTodoDescription = e.target.value} />
```

Comment:
As we have removed newTodoDescription variable, we are controlling the input by useState.
```suggestion
        <input type='text' onChange={(e) => setUserInput(e.target.value)} />
```